### ISSUE TITLE
Add three body problem functions

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#8]
And an example to show these capabilities.

**Comment by Juanlu001** [comment]
Blocked on #23.

**Comment by Juanlu001** [comment]
On scope, though vague and probably complicated.

**Comment by Juanlu001** [comment]
Out of scope? Perhaps starting with proper sphere of influence and patched conics computations would be more appropriate. Opening a new issue and leaving this as an open question.

**Comment by Juanlu001** [comment]
Sphere of influence: #150 
Lagrange points: #173 

This is a non issue, so closing!
